<launch>
   <include file="$(find nuslam)/launch/movement.launch">
   </include>
   <rosparam command="load" file="$(find nuslam)/config/slam_params.yaml" />  

   <include file="$(find nuturtle_gazebo)/launch/diff_drive_gazebo.launch">
   </include>

   <!-- <node name="analysis" pkg="nuslam" type="analysis" >
   <remap from="landmarks" to="real/landmarks"/>  
   </node>  -->

   <!-- <node name="slam" pkg="nuslam" type="slam" output = "screen">
   <remap from="landmarks" to="real/landmarks"/>  
   </node> -->

   <node name="draw_map" pkg="nuslam" type="draw_map"/>


   <!-- <node name="draw_map_real" pkg="nuslam" type="draw_map">
   <remap from="landmarks" to="real/landmarks"/>  
   <remap from="visualization_marker_scan_array" to="real/visualization_marker_scan_array"/>  

   </node> -->

   
   <node  name="draw_map_slam" pkg="nuslam" type="draw_map">
   <remap from="landmarks" to="slam/landmarks"/>  
   <remap from="visualization_marker_scan_array" to="slam/visualization_marker_scan_array"/>  
   </node>
   
</launch>

